src/reducers/index.js

```diff
@@ -1,37 +1,4 @@
-const dbReducer = (state = {}, action) => {
-  switch(action.type){
-    case 'GET_LOCATION_RESPONSE':
-      if(action.payload){
-        return {...state, [action.payload.id] : action.payload}
-      }
-      break;
-    case 'OPTIMISTIC_UPDATE_LOCATION':
-      if(action.payload){
-        return {
-          ...state,
-          [`last/${action.payload.id}`] : state[action.payload.id],
-          [action.payload.id] : { 
-            ...state[action.payload.id],
-            ...action.payload.params,
-          }
-        }
-      }
-      break;
-    case 'ROLLBACK_UPDATE_LOCATION':
-      if(action.payload){
-        return {
-          ...state,
-          [`last/${action.payload.id}`] : null,
-          [action.payload.id] : state[`last/${action.payload.id}`]
-        }
-      }
-      break;
-    default:
-      return state;
-  }
-
-  return state;
-}
+import dbReducer from './db';
 
 export const selectLocationData = (state, locationId) => state.db[locationId];
```

Comment:
This probably belongs in the db reducer.

Reply:
Fixed in b5a14de702a74df30597583ec35db824963a009a